.node-version

```diff
@@ -1 +1 @@
-16.18.0
+18.18.0
```

Comment:
Bumping this would be cleaner to do in a different PR if possible (even though it's a good idea) since it's a bigger change and not needed for this ticket.